webtau-feature-testing/src/main/groovy/com/twosigma/webtau/featuretesting/WebTauEndToEndTestRunner.groovy

```diff
@@ -129,6 +129,16 @@ class WebTauEndToEndTestRunner implements StepReporter, StandaloneTestListener {
         }
     }
 
+    private static Map sortTestDetailsByContainerId(Map testDetails) {
+        def scenarioDetails = testDetails.scenarioDetails
```

Comment:
should we make a copy of list or explicitly modify in place? RIght now it makes it look like it returns a copy, but it doesn't, does it?

Reply:
Bit of both :)  It creates a sorted copy of the list but then overrides it in the main map.  I return the map just to make the calling code easier.

Having said that, I can see why it's confusing.  I'll make it all copy.